.drone.env

```diff
@@ -1,3 +1,3 @@
 # The test runner source for UI tests
-WEB_COMMITID=a26f7920d4ef8bf57488b18e451525d16e246e9e
-WEB_BRANCH=stable-7.1
+WEB_COMMITID=5bdcb9671c43a7f0e2e0dc1c4f6f0584982dc234
+WEB_BRANCH=sevenone-to-master
```

Comment:
Shouldn't that be master ?

Reply:
it should, but i wanna see that it is green here before we merge web to master... hust in case, once it is green, ill change it back to master

Reply:
This commit ID is only to get the tests from web, we don't pull the web assets from that branch/commit. We want to test the web version that is currently build-in into ocis.
see also https://github.com/owncloud/ocis/pull/7198#issuecomment-1704607164